src/popup/sass/layout/_search.scss

```diff
@@ -22,17 +22,21 @@
   i {
     position: absolute;
     right: 0;
```

Comment:
Remove `right: 0`. This was also written in the comment above .

Reply:
Done. I'm really sorry for all the tiny errors i made